src/pages/userPage/Secretaries/UserGoverningBodySecretaryTable.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useEffect, useState} from 'react';
+import {Table, Empty} from 'antd';
+import columns from './columnsGoverningBodies';
+import {
+    getUsersAdministrations,
+    getUsersPreviousAdministrations
+} from "../../../api/governingBodiesApi";
+import Modal from "antd/lib/modal";
+import SecretaryModel from './SecretaryModel';
+
+interface props {
+    UserId: string;
+}
+
+export const UserGoverningBodySecretaryTable = ({UserId}: props) => {
+    const [isLoadingActive, setIsLoadingActive] = useState<boolean>(true);
+    const [isLoadingPrev, setIsLoadingPrev] = useState<boolean>(true);
+
+    const [data, setData] = useState<SecretaryModel[]>();
```

Comment:
data -should be called GoverningBodyAdmins

Reply:
done